src/frontend/Komponenter/Personoversikt/BehandlingsoversiktTabell.tsx

```diff
@@ -117,6 +117,21 @@ export const BehandlingsoversiktTabell: React.FC<{
         return `${tilbakekrevingBaseUrl()}/fagsystem/EF/fagsak/${eksternFagsakId}/behandling/${behandlingId}`;
     };
 
+    const lagKlagebehandlingsLenke = (behandlingId: string): string => {
+        return `${klageBaseUrl()}behandling/${behandlingId}/formkrav`;
```

Comment:
Her mangler det kanskje en `/`? 
```suggestion
        return `${klageBaseUrl()}/behandling/${behandlingId}/formkrav`;
```

Reply:
Trenger vi `/formkrav` på sluttet, eller er det tilstrekkelig å kun sende til `/${behandlingId}` ?